sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -59,6 +59,165 @@ public DigitalTwinsServiceVersion getServiceVersion() {
+    {
+        return digitalTwinsAsyncClient.createDigitalTwinWithResponse(digitalTwinId, digitalTwin, clazz, context).block();
+    }
+
+    /**
+     * Gets a digital twin.
+     *
+     * @param digitalTwinId The Id of the digital twin. The Id is unique within the service and case sensitive.
+     * @return The application/json digital twin
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public String getDigitalTwin(String digitalTwinId)
+    {
+        return getDigitalTwinWithResponse(digitalTwinId, Context.NONE).getValue();
+    }
+
+    /**
+     * Gets a digital twin.
+     *
+     * @param digitalTwinId The Id of the digital twin. The Id is unique within the service and case sensitive.
+     * @param context Additional context that is passed through the Http pipeline during the service call.
+     * @return A Http response containing application/json digital twin
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<String> getDigitalTwinWithResponse(String digitalTwinId, Context context)
+    {
+        return digitalTwinsAsyncClient.getDigitalTwinWithResponse(digitalTwinId, context).block();
+    }
+
+    /**
+     * Gets a digital twin.
+     *
+     * @param digitalTwinId The Id of the digital twin. The Id is unique within the service and case sensitive.
+     * @param clazz The model class to convert the response to.
+     * @return The application/json digital twin
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public <T> T getDigitalTwin(String digitalTwinId, Class<T> clazz)
+    {
+        return getDigitalTwinWithResponse(digitalTwinId, clazz, Context.NONE).getValue();
+    }
+
+    /**
+     * Gets a digital twin.
+     *
+     * @param digitalTwinId The Id of the digital twin. The Id is unique within the service and case sensitive.
+     * @param clazz The model class to convert the response to.
+     * @param context Additional context that is passed through the Http pipeline during the service call.
+     * @return A Http response containing application/json digital twin
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public <T> Response<T> getDigitalTwinWithResponse(String digitalTwinId, Class<T> clazz, Context context)
+    {
+        return digitalTwinsAsyncClient.getDigitalTwinWithResponse(digitalTwinId, clazz, context).block();
+    }
+
+    /**
+     * Updates a digital twin.
+     *
+     * @param digitalTwinId The Id of the digital twin.
+     * @param digitalTwinUpdateOperations The application/json-patch+json operations to be performed on the specified digital twin
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public void updateDigitalTwin(String digitalTwinId, List<Object> digitalTwinUpdateOperations)
+    {
+        updateDigitalTwinWithResponse(digitalTwinId, digitalTwinUpdateOperations, new RequestOptions(), Context.NONE).getValue();
```

Comment:
You don't need to call `.getValue()` here, since the API neither does anything with the value, nor does it return it.
You can invoke the API and leave it at that.

Reply:
Good point

Reply:
Interestingly, the build pipelines will fail if you were to leave this calling ```.getValue()``` since the bug detector thinks we are ignoring a return type when we use it like this. I ran into this on the component APIs as well.

Reply:
There have very thorough checks at the gate!